Assets/UI/Scripts/PanelPager.cs

```diff
@@ -19,12 +19,19 @@ public class PanelPager : MonoBehaviour
     public Button PreviousButton = null;
     /// <summary>Button to move to the next panel</summary>
     public Button NextButton = null;
+    /// <summary>Button to complete onboarding and exit to the next scene</summary>
+    public Button DoneButton = null;
 
     /// <summary>Index of the currently active panel</summary>
     private int currentIndex = 0;
 
     void Start()
     {
+        // Attches respective button variables to unity buttons
+        PreviousButton = GameObject.Find("PreviousButton").GetComponent<Button>();
+        NextButton = GameObject.Find("NextButton").GetComponent<Button>();
+        DoneButton = GameObject.FindWithTag("DoneButton").GetComponent<Button>();
```

Comment:
These values are set through the Unity Inspector UI so the script doesn't need to set them.

Reply:
It wasn't setting them up the right way. That was the issue with the Previous Button before -- it assigned the unity NextButton to the script PreviousButton and was causing a lot of problems (and then the same thing happened with my DoneButton). I don't know how to fix that in unity, so I just did it in the script. If you know another way to fix it, please tell me :)

Reply:
Although I just realized I may have committed the wrong file as it should just be Find, not FindWithTag for the last one (I was playing around in another document to experiment with some things). My bad, I'll fix that right now.